### DISCUSSION TITLE
Pensela on i3-gaps

### DISCUSSION THREAD

**Discussion by pablogzlezmora** [weiameili/Pensela#20] [Q&A] [answered]
How to use Pensela on i3-wm/i3-gaps?

I can't “overlap” the Pensela window on the working window. :-(

**Comment by pablogzlezmora** [comment]
Some good person suggested me this thread. 
https://github.com/weiameili/Pensela/issues/8
I am going to read it.

**Comment by weiameili** [comment, marked as answer]
Hi, although adding `for_window [class="pensela"] floating enable` allows it to overlap, unfortunately there are other issues because of which Pensela is unusable on i3 (see the issue you linked above)

**Reply by pablogzlezmora** [reply]
I've seen the issue. Thanks for your answer.
